packages/plugin-core/src/commands/pods/GoogleDocsExportPodCommand.ts

```
@@ -174,44 +175,49 @@ export class GoogleDocsExportPodCommand extends BaseExportPodCommand<
 
   async onExportComplete(opts: {
     exportReturnValue: GoogleDocsExportReturnType;
-    payload: NoteProps;
+    payload: string | NoteProps[];
     config: RunnableGoogleDocsV2PodConfig;
   }) {
     const { exportReturnValue, payload } = opts;
+    const createdDocs = exportReturnValue.data?.created?.filter((ent) => !!ent);
+    const updatedDocs = exportReturnValue.data?.updated?.filter((ent) => !!ent);
+    const createdCount = createdDocs?.length ?? 0;
+    const updatedCount = updatedDocs?.length ?? 0;
+    if (typeof payload !== "string" && createdDocs && createdCount > 0) {
+      await this.updateNoteWithCustomFrontmatter(createdDocs);
+    }
+    if (typeof payload !== "string" && updatedDocs && updatedCount > 0) {
+      await this.updateNoteWithCustomFrontmatter(updatedDocs);
+    }
+
     if (ResponseUtil.hasError(exportReturnValue)) {
-      this.L.error(exportReturnValue.error);
-      window.showErrorMessage(
-        "Error while running Google Docs Export Pod: " + exportReturnValue
+      const errorMsg = `Finished GoogleDocs Export. ${createdCount} docs created; ${updatedCount} docs updated. Error encountered: ${ErrorFactory.safeStringify(
```

can you return this error and also add a test for this?